hub/generalities/exceptions/vanishing.tex

```diff
@@ -1,17 +1,42 @@
 \begin{center}
-	\boxed{\text{All constraints in this section are written under the assumption } \peekStack_{i} = 1.}
+	\boxed{\text{All constraints in this section are written under the assumption } \peekStack _{i} = 1.}
 \end{center}
 What follows are constraints preventing certain instruction families from triggering certain exceptions.
 Thus reducing the complexity of the ``exception analysis'' required in the instruction handling.
 \begin{enumerate}
-	\item $\stackOpcx    _{i} = \stackDecInvalidFlag_{i}$
-	\item \If $\stackDecMxpFlag_{i}     = 0$ \Then $\stackMxpx    _{i} = 0$
-	\item \If $\stackDecJumpFlag_{i}    = 0$ \Then $\stackJumpx   _{i} = 0$
-	\item \If $\decStaticInst_{i}  = 0$ \Then $\stackStaticx _{i} = 0$
-	\item \If $\stackInst_{i} \neq \inst{RETURNDATACOPY}$ \Then $ \stackRdcx_{i}    = 0 $
-	\item \If $\stackInst_{i} \neq \inst{SSTORE}$         \Then $ \stackSstorex_{i} = 0 $
-	\item \If $\stackInst_{i} \neq \inst{RETURN}$         \Then $ \stackIcpx_{i} + \stackMaxcsx_{i} = 0 $
+	\item
+		$\stackOpcx _{i} = \stackDecInvalidFlag _{i}$
+	\item
+		\If   $\stackDecMxpFlag _{i} = 0$
+		\Then $\stackMxpx       _{i} = 0$
+	\item
+		\If   $\stackDecJumpFlag _{i} = 0$
+		\Then $\stackJumpx       _{i} = 0$
+	\item
+		\If   $\decStaticInst _{i} = 0$
+		\Then $\stackStaticx  _{i} = 0$
+	\item
+		\If   $\stackInst _{i} \neq \inst{RETURNDATACOPY}$
+		\Then $\stackRdcx _{i} =    0$
+	\item
+		\If   $\stackInst    _{i} \neq \inst{SSTORE}$
+		\Then $\stackSstorex _{i} =    0$
+	\item
+		\If   $\stackInst _{i} \neq \inst{RETURN}$
+		\Then $\stackIcpx _{i} =    0$
+	\item
+		\If   $\stackInst   _{i} \neq \inst{RETURN}$ \et $\stackCreateFlag _{i} = 0$
+		\Then $\stackMaxcsx _{i} =    0$
```

Comment:
Previously only RETURN could trigger `invalidCodePrefixException` and `maxCodeSizeException`. Now `CREATE(2)` can trigger `maxCodeSizeException`, too. So split analysis.